app/services/user_creation_service.rb

```diff
@@ -0,0 +1,45 @@
+# dead simple logger. Expect to see a fake version that's cleverer in specs
```

Comment:
My thought here was to explain why I'm not just putting `puts` directly in the service, but upon retrospection that might be self-evident from the dependency injection in the initalize